src/ui/display.py

```
@@ -18,35 +19,31 @@
+
     for entry in feed.entries[:5]:
         # Get the time struct from either published or updated date
-        time_struct = entry.get('published_parsed') or entry.get('updated_parsed')
-        
+        time_struct = entry.get("published_parsed") or entry.get("updated_parsed")
+
         if time_struct:
             # Convert time_struct to Unix timestamp then to datetime
             timestamp = time.mktime(time_struct)
-            date = datetime.fromtimestamp(timestamp).strftime('%Y-%m-%d')
+            date = datetime.fromtimestamp(timestamp).strftime("%Y-%m-%d")
```

Ruff (0.8.0)

`datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)